routes/candidates.js

```diff
@@ -1,71 +1,97 @@
 import models from '../models'
+import userIsAdmin from '../lib/userIsAdmin'
 import express from 'express'
 
 const router = express.Router()
 
 router.get('/:candidateId?', (req, res) => {
-	models.Candidate.findAll({
-		where: {
-			id: (req.params.candidateId == null) ? '*' : req.params.candidateId
```

Comment:
Found out that MySQL doesn't allow `*` on integer columns for some reason, so I transitioned to a much uglier syntax for all the routes. If someone has a better way to do this, I'd love to hear it.